### PR TITLE
Home page edits v1

### PR DESCRIPTION
**Pull request by valeglezleyva**
HomePage

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -22,8 +22,18 @@
     -->
   <title>Secret Santa App</title>
 
+
```
Comment (ystamaritq) [inline client/public/index.html:25]: This was removed in the previous PRs, please remove it! Thanks

**Review by ystamaritq — CHANGES_REQUESTED** [review]

**Review by JDMartinez1531 — APPROVED** [review]

**Review by ystamaritq — COMMENTED** [review]
Looks great! But, I will approve as soon as code-climate passes. Thanks 

image

**Review by KEDuran — DISMISSED** [review]
Hi Vale :) Great job! Code climate is still not passing. Please refactor and then I will approve. Thanks!

**Review by ystamaritq — APPROVED** [review]
Looks great! Great job. Issues resolved

**Pull request merged**